spec/document/documents_spec.rb

```diff
@@ -16,6 +16,73 @@ def read_file(dir_path, file_name)
     File.read(File.join(dir_path, file_name)).strip
   end
 
+  context 'A FinancialDocumentV1' do
+    context 'when processing an invoice' do
+      it 'should load an empty document prediction' do
+        response = load_json(DIR_FINANCIAL_DOCUMENT_V1, 'empty.json')
+        document = Mindee::Document.new(Mindee::Prediction::FinancialDocumentV1, response['document'])
+        expect(document.inference.product.type).to eq('standard')
+        prediction = document.inference.prediction
+        expect(prediction.invoice_number.value).to be_nil
+        expect(prediction.invoice_number.polygon).to be_empty
+        expect(prediction.invoice_number.bounding_box).to be_nil
+        expect(prediction.date.value).to be_nil
+        expect(prediction.date.page_id).to be_nil
```

Comment:
This is failing because with the test data return 0 instead of nil, not sure which one is the correct